Formula/mas.rb

```diff
@@ -14,11 +14,34 @@ class Mas < Formula
   depends_on :xcode => ["9.0", :build]
 
   def install
-    xcodebuild "-project", "mas-cli.xcodeproj",
+    # Pre-install a shallow copy of the CocoaPods master repo
+    system "git", "clone", "--depth", "1", "https://github.com/CocoaPods/Specs.git", File.expand_path("~/.cocoapods/repos/master")
+
+    # Install bundler, then use it to install gems used by project
+    ENV["GEM_HOME"] = buildpath/"gem_home"
+    system "gem", "install", "bundler"
+    ENV.prepend_path "PATH", buildpath/"gem_home/bin"
+    system "bundle", "install"
+    system "bundle", "exec", "pod", "install"
+
+    xcodebuild "-workspace", "mas-cli.xcworkspace",
                "-scheme", "mas-cli Release",
-               "-configuration", "Release",
-               "SYMROOT=build"
-    bin.install "build/mas"
+               "SYMROOT=#{File.absolute_path(buildpath)}"
+
+    bin.install buildpath/"build/mas"
+
+    bash_completion.install "contrib/completion/mas-completion.bash" => "mas"
+  end
+
+  def caveats; <<~EOS
```

Comment:
I'd rather not include these caveats, personally, as much as I appreciate that may be frustrating for y'all. We try to limit caveats pretty hard because, well, nobody really reads them and it doesn't help there are so many already 🙃.

Reply:
gotcha, I'll 🔥 it